concourse/pipelines/cd-frontend.yml

```
@@ -0,0 +1,42 @@
+jobs:
+- name: update-pipeline
+  plan:
+  - get: govuk-infrastructure
+    trigger: true
+  - file: govuk-infrastructure/concourse/pipelines/cd-frontend.yml
+    set_pipeline: govuk-cd-frontend
+- name: deploy
+  plan:
+  - get: govuk-infrastructure
```

```suggestion
  - get: govuk-infrastructure
    passed: [update-pipeline]
```

If your change contains an update to the pipeline, and an update to the infrastructure, they should happen in a deterministic fashion.

---

Thanks!